DXMainClient/DXGUI/Generic/LoadingScreen.cs

```diff
@@ -40,22 +40,32 @@ public override void Initialize()
             ClientRectangle = new Rectangle(0, 0, 800, 600);
             Name = "LoadingScreen";
 
-            string backgroundName = "loadingscreen.png";
+            string GetBackgroundName(int id)
```

Comment:
AFAIK local functions are a feature of C# 7.0 which is unsupported by XNA build of the client which builds against .NET Framework 4.0. Should make it a regular function and add an access modifier, assumingly `private` here.

Reply:
I'm not facing any problem when compiling all the three builds with Visual Studio 2019. The `clientxna.exe` file is generated correctly. Anyway, I have updated the code.